PSKoans/Classes/Blank.ps1

```diff
@@ -0,0 +1,9 @@
+class Blank {
+    [string] ToString() {
+        return $null
+    }
+    
+    [bool] op_Equals([object] $other) {
+        return $false
+    }
+}
```

Comment:
Last thing (I promise!)

Please make sure there's a newline at the end of the file here. 🙂

Reply:
Usually my VS code does this for me, must have lost the setting somewhere. Fixed.